data/com.github.geigi.cozy.appdata.xml

```diff
@@ -42,19 +44,16 @@
       <image>https://raw.githubusercontent.com/geigi/cozy/img/img/screenshot4.png</image>
     </screenshot>
   </screenshots>
-  <developer_name>Julian Geywitz</developer_name>
+  <developer_name translatable="no">Julian Geywitz</developer_name>
   <url type="homepage">https://cozy.sh</url>
   <url type="bugtracker">https://github.com/geigi/cozy/issues</url>
-  <url type="help">https://github.com/geigi/cozy/issues</url>
```

Comment:
I think it should point to [`https://matrix.to/#/#cozy:gnome.org?via=matrix.org&via=gnome.org`](https://matrix.to/#/#cozy:gnome.org?via=matrix.org&via=gnome.org) instead